src/orders/components/OrderUnfulfilledProductsCard/OrderUnfulfilledProductsCard.tsx

```diff
@@ -20,6 +20,13 @@ interface OrderUnfulfilledProductsCardProps {
   onShowMetadata: (id: string) => void;
 }
 
+const toLineWithUnfulfilledQuantity = (lines: OrderLineFragment[]) =>
+  lines.map(({ quantityToFulfill, ...rest }) => ({
+    ...rest,
+    quantityToFulfill,
+    quantity: quantityToFulfill,
+  }));
```

Comment:
Any test, any information why you did it

Reply:
I'll add a comment, but I'm not sure there's any meaningful test to add here